### PR TITLE
Add new zones to AWS

### PR DESCRIPTION
**Pull request by bathina2**
## Change Overview

The following PR add the new missing zones to AWS.

## Pull request type

Please check the type of change your PR introduces:
- [ ] :construction: Work in Progress
- [ ] :rainbow: Refactoring (no functional changes, no api changes)
- [x] :hamster: Trivial/Minor
- [ ] :bug: Bugfix
- [ ] :sunflower: Feature
- [ ] :world_map: Documentation
- [ ] :robot: Test

## Issues

- #XXX

## Test Plan

- [ ] :muscle: Manual
- [ ] :zap: Unit test
- [ ] :green_heart: E2E

### CODE REVIEW

**Review by TribalNightOwl — APPROVED** [review]

File: pkg/blockstorage/awsebs/awsebs.go [hunk 1]
```diff
@@ -736,6 +736,9 @@ func staticRegionToZones(region string) ([]string, error) {
 			"eu-central-1a",
 			"eu-central-1b",
 			"eu-central-1c",
+			"eu-central-1-wl1-ber-wlz-1",
```
Comment (julio-lopez) [inline pkg/blockstorage/awsebs/awsebs.go:739]: That's an interesting naming scheme.
Is there an official documentation link for these?

Comment (bathina2) [reply pkg/blockstorage/awsebs/awsebs.go:739]: The names are defined by AWS. We call the list regions API and these are the values it provides.

Comment (julio-lopez) [reply pkg/blockstorage/awsebs/awsebs.go]: This is what the AWS CLI spews out:

```
aws ec2 describe-availability-zones --region eu-central-1 |jq .AvailabilityZones[].ZoneName

"eu-central-1a"
"eu-central-1b"
"eu-central-1c"

```

Comment (bathina2) [reply pkg/blockstorage/awsebs/awsebs.go]: @julio-lopez The result varies by the credentials I think. Another reason why this should be dynamic.

Comment (julio-lopez) [reply pkg/blockstorage/awsebs/awsebs.go]: x-reference: 

```
aws ec2 describe-availability-zones --all-availability-zones --region eu-central-1 |jq .AvailabilityZones[].ZoneName
"eu-central-1a"
"eu-central-1b"
"eu-central-1c"
"eu-central-1-wl1-ber-wlz-1"
"eu-central-1-wl1-dtm-wlz-1"
"eu-central-1-wl1-muc-wlz-1"
```

**Review by julio-lopez — APPROVED** [review]

**Pull request merged**